tokio/src/net/addr.rs

```diff
@@ -14,12 +14,11 @@ use std::net::{IpAddr, Ipv4Addr, Ipv6Addr, SocketAddr, SocketAddrV4, SocketAddrV
 /// # Calling
 ///
 /// Currently, this trait is only used as an argument to Tokio functions that
-/// need to reference a target socket address.
+/// need to reference a target socket address. To perform a `SocketAddr`
```

Comment:
Not directly related to the change, but adding `FromStream` made me think to tweak this documentation.